org.scala-ide.sdt.core.tests/src/org/scalaide/core/ui/CommentAutoEditStrategyTest.scala

```diff
@@ -104,6 +104,24 @@ class CommentAutoEditStrategyTest extends AutoEditStrategyTests {
   }
 
   @Test
+  def openDocComment_topLevel_with_stringLitCommon() {
+    """
+    /**
+     * ""^
+    class Foo {
```

Comment:
I don't know if I understand this test case correctly, but issue states "Closed multi line comment is closed again". 

This test looks like it only checks if it closes correctly the first time.